ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -7,6 +7,181 @@
   },
   timelineFile: 'e5s.txt',
   triggers: [
+    {
+      id: 'E5S Fury\'s Bolt Gain',
+      regex: Regexes.gainsEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
+      regexDe: Regexes.gainsEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
+      regexFr: Regexes.gainsEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
+      regexJa: Regexes.gainsEffect({ target: 'ラムウ', effect: 'チャージボルト', capture: false }),
+      run: function(data) {
+        data.fury = true;
+      },
+    },
+    {
+      id: 'E5S Fury\'s Bolt Lose',
+      regex: Regexes.losesEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
+      regexDe: Regexes.losesEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
+      regexFr: Regexes.losesEffect({ target: 'Ramuh', effect: 'Fury\'s Bolt', capture: false }),
+      regexJa: Regexes.losesEffect({ target: 'ラムウ', effect: 'チャージボルト', capture: false }),
+      run: function(data) {
+        data.fury = false;
+      },
+    },
+    {
+      id: 'E5S Stratospear Summons',
+      regex: Regexes.ability({ id: '4BA5', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.ability({ id: '4BA5', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BA5', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BA5', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BA5', source: '라무', capture: false }),
+      delaySeconds: 5,
+      condition: function(data) {
```

Comment:
I may be misreading, but is this ever going to fire? I couldn't find another reference to `seenFirstSpear` outside this trigger. If I'm remembering the order of operations correctly, the condition won't fire, and without that the run() command to set the value won't either. I think it might instead need to be

`return !data.seenFirstSpear`

Reply:
yes, your're right. if you haven't seen this, trigger should do some work. (when `seenFirstSpear` returns false